backend/TreeOfAKind.Domain/UserProfiles/UserProfile.cs

```diff
@@ -0,0 +1,66 @@
+﻿using System;
+using System.Globalization;
+using System.Threading;
+using NodaTime;
+using TreeOfAKind.Domain.SeedWork;
+
+namespace TreeOfAKind.Domain.UserProfiles
+{
+    public class UserProfile : Entity, IAggregateRoot
```

Comment:
There probably should be an email in here.

Reply:
We will have emails in Firebase, we will ask for it by calling firebase auth API, by providing authUserId.

If we kept emails in the domain we would have to have triggers in firebase that will update our domain on email changed.
It is somewhat a shortcut but is not our core domain to store multiple emails for notifications.